_includes/components/slider.html

```diff
@@ -1,27 +1,30 @@
-{% assign event = include.event %}
-{% assign impressions = nil | arrayify %}
-{% for file in site.static_files %}
-  {% assign isImpression = file.path | startswith: "/resources/images/impressions/" %}
-  {% if isImpression %}
-    {% assign fileArray = file | arrayify %}
-    {% assign impressions = impressions | concat: fileArray %}
-  {% endif %}
-{% endfor %}
-{% if impressions.first %}
+{% capture imageQuery %}
+  SELECT ?image ?description ?link
+  WHERE {
+    ?image rdf:type foaf:Image ;
+      rdf:value ?link ;
+      dct:description ?description .
+    FILTER(langMatches(lang(?description), "{{ site.language }}"))
+    #FILTER(strstarts(?link, "/resources/images/impressions/"))
+  }
+{% endcapture %}
+{% assign images = rootResource | sparql_query: imageQuery %}
+{% if images.first %}
   <div class="slider-box">
     <div class="slider">
       <ul>
-        {% for impression in impressions %}
+        {% for image in images %}
           <li>
-            <img class="js-load" data-src="{{ impression.path | relativize_url }}" data-sizes="auto"
-              data-srcset="{{ impression.path | relativize_url }} 800w, {{ impression.path | relativize_url }} 1980w"
-              width="6240" height="4160" style="max-width:6240px" alt="">
+            <img class="js-load" data-src="{{ image.link }}" data-sizes="auto"
```

Comment:
please also relativize the paths here and better use the img include https://github.com/AKSW/leipzig.dataweek.de/blob/develop/_includes/components/img.html
since these images don't make use of the various widths anyhow.

(Maybe this is also a cause for https://github.com/AKSW/leipzig.dataweek.de/issues/117.)

Reply:
Paths are relative now.
Image auto sizing is working I guess, but we use the same file for the two given widths. 
Also including the image component won't help, since it is not variable regarding the widths at the moment. (Set to 120)

Reply:
You could extend the img component accordingly, that would help in many places.

Reply:
Yes, I think this should be a separate issue and we should discuss the naming convention for filenames in the next meeting.

Reply:
New issue: #120